src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.domain;
+
+import racingcar.enums.Movement;
+
+public class Car {
+    private final String name;
+    private int position = 0;
+
+    public Car(String name) {
+        this.name = name;
+    }
```

Comment:
```suggestion
    private final String name;
    private int position;

    public Car(String name) {
        this.name = name;
        this.position = 0;
    }
```
Car라는 객체의 속성을 명시적으로 보여주기 위해, 그리고 position의 기본값이 Car 객체가 생성될 때 함께 초기화 되는 효과를 위해서 필드 주입으로 0으로 초기화하는 방법보다, 생성자에 position의 초기화를 넣는 건 어떨까요?

Reply:
맞네요 !
객체 생성시 마다 위치 값이 초기화 되는 게 훨씬 가독성이 좋아보입니다!